app/src/main/java/com/epam/caloriecalc/ui/addnew/AddItemScreen.kt

```diff
@@ -0,0 +1,83 @@
+package com.epam.caloriecalc.ui.addnew
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.epam.caloriecalc.R
+import com.epam.caloriecalc.data.local.entities.ProductRecord
+import com.epam.caloriecalc.util.AddItemEvent
+import com.epam.caloriecalc.util.UiEvent
+import kotlinx.coroutines.flow.collect
+
+
+@Composable
+fun AddItemScreen(
+    viewModel: AddItemViewModel = hiltViewModel(),
+    itemlist: List<ProductRecord>
+) {
+
+    val scaffoldState = rememberScaffoldState()
+
+    val context = LocalContext.current
```

Comment:
let's try to avoid keeping a context reference, especially if it's totally unnecessary, you can use stringResource() instead of getString()

Reply:
never mind, I have realized it's not possible